typing/src/main/java/org/navgurukul/typing/dashboard/advance/AdvanceFragment.kt

```diff
@@ -0,0 +1,28 @@
+package org.navgurukul.typing.dashboard.advance
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import org.navgurukul.typing.R
+
+class AdvanceFragment : Fragment() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+    }
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        // Inflate the layout for this fragment
+        return inflater.inflate(R.layout.fragment_advance, container, false)
+    }
+
+    companion object {
+        @JvmStatic
```

Comment:
Is this intentional? Fragments should ideally be recreated when the activity is recreated.